### ISSUE TITLE
" to a Minimum of -30" means don't go any lower?

### ISSUE THREAD

**Issue by Memnarch** [PrimeO7/How-to-undervolt-AMD-RYZEN-5800X3D-Guide-with-PBO2-Tuner#13]
Huhu,
Just needs some clarification. Going from -10 in steps of -5 to a minimum of -30. Does this mean -30 is the most sane someone should go and not go lower, or does it mean -30 is something most should be able to reach safely.

**Comment by PrimeO7** [comment]
You can't go lower than -30. You can type in this offset, but it won't apply it.
Considering safety. All negative offsets are "safe".
You won't damage your hardware, but might encounter system instability if you go to low. 
Most 5800X3D can go pretty low. I've been running -30 on all cores for months now without a single crash

**Comment by Memnarch** [comment]
Ah ok.
For context:
I recently replaced my R5 3600 with the 5800x3D. I had an AIO around from the Intel days (Corsair H60) but that one hadn't been used for 3 years now (and itself was like 7 years old probably) and I was missing the AMD Socket brackets. So I simply reused the stockcooler from the R5 3600 (which does not even have a copper plate).

Stock settings, single core could boost to 4.5GHZ without problems but AllCore Boost would shrink down till it reached 3.8GHZ, for obvious reasons.

With -30 on the tuner, I now get a stable 4GHZ allcoreboost on my system for a Cinebench R23 10 Minute run (even multiple in a row) while it does not seem to have negatively influenced the single core performance. So that's what I'll run with for now, even when I get myself a new cooler next year or so^^.

Old stock MP Score was something around 12200 (but would go up to 12600 at times when running multiple 10 minute runs...maybe to much paste? Though probably now)

Now the mp score is 13850 and for an all core boost I am fine with it i think.
PS: For reference, the single core score is 1485

PSS: After having the scheduler stuff setup and a clean reboot, the system amanged its best MP run with 13949.
